php/helpers.php

```diff
@@ -89,6 +89,8 @@ function block_field( $name, $echo = true ) {
 		 * and then output the field with a more suitable escaping function.
 		 */
 		echo wp_kses_post( $value );
+
+		return null;
```

Comment:
We should probably add `@return mixed` to the docblock.

Reply:
OK, add it to the `block_field()` DocBlock? It looks like it's already there, but I could have misunderstood this.

Reply:
Ah my bad I was looking at the wrong lines, sorry!

Reply:
All good, thanks for reviewing this